### PR TITLE
bumping versions of python(2) tools

### PR DESCRIPTION
**Pull request by davidlange6**
More complicated updates to come via separate PR

### CODE REVIEW

**Comment by davidlange6** [comment]
please test


On Jul 23, 2020, at 1:22 PM, David Lange <notifications@github.com<mailto:notifications@github.com>> wrote:


More complicated updates to come via separate PR

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/cms-sw/cmsdist/pull/6108

Commit Summary

  *   bumping versions of py2 tools

File Changes

  *   M pip/requirements.txt<https://github.com/cms-sw/cmsdist/pull/6108/files#diff-8d33d96e5ae1b0fd471d1a8fa5d1ff6b> (112)

Patch Links:

  *   https://github.com/cms-sw/cmsdist/pull/6108.patch
  *   https://github.com/cms-sw/cmsdist/pull/6108.diff

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
ah - more fun with python2 support being dropped.ok. fixing again

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
seems that these recotau differences are also seen in other pull requests that are purely technical. A known issue? @smuzaffar

**Comment by smuzaffar** [comment]
no idea. Normally we get false positive due to MessageLogger but these does not look like ML differences.

**Comment by davidlange6** [comment]
I agree. They are also found in your gdb pr for example


On 28 Jul 2020, at 09:14, Malik Shahzad Muzaffar <notifications@github.com> wrote:

﻿

no idea. Normally we get false positive due to MessageLogger but these does not look like ML differences.

File: pip/requirements.txt [hunk 1]
```diff
@@ -140,9 +141,9 @@ networkx==2.4 ; python_version>'3.0'
 neurolab==0.3.5
 nose-parameterized==0.6.0
 nose==1.3.7
-notebook==5.7.8 ; python_version<'3.0'
+notebook==5.7.10 ; python_version<'3.0'
 notebook==6.0.2 ; python_version>'3.0'
-numba==0.50.0dev0 ; python_version>'3.0'
+numba==0.50.1 ; python_version>'3.0'
```
Comment (smuzaffar) [inline pip/requirements.txt:146]: @davidlange6 , numba 0.50.1 was not picked up due to hard coded commit in the https://github.com/cms-sw/cmsdist/blob/d31f5a3c35fca2d12b62afe01eb6441f6840b126/pip/py3-numba.file .

@mrodozov , please check if py3-numba-llvm33  patch is needed for numba 0.50.1?

**Comment by davidlange6** [comment]
Ah - I missed that.

That commit is part of 0.50.1 so we can remove this. I’ll do it.

On Jul 28, 2020, at 1:05 PM, Malik Shahzad Muzaffar <notifications@github.com<mailto:notifications@github.com>> wrote:



@smuzaffar commented on this pull request.

________________________________

In pip/requirements.txt<https://github.com/cms-sw/cmsdist/pull/6108#discussion_r461498315>:

>  notebook==6.0.2 ; python_version>'3.0'
-numba==0.50.0dev0 ; python_version>'3.0'
+numba==0.50.1 ; python_version>'3.0'


@davidlange6<https://github.com/davidlange6> , numba 0.50.1 was not picked up due to hard coded commit in the https://github.com/cms-sw/cmsdist/blob/d31f5a3c35fca2d12b62afe01eb6441f6840b126/pip/py3-numba.file .

@mrodozov<https://github.com/mrodozov> , please check if py3-numba-llvm33 patch is needed for numba 0.50.1?

**Comment by davidlange6** [comment]
Sorry, I was not understanding the situation fully - so no, this patch is not in 0.50.1 or master of Numba. As I don’t understand why we need it I’ll try removing it to run a test.

On Jul 28, 2020, at 2:24 PM, David Lange <notifications@github.com<mailto:notifications@github.com>> wrote:


Ah - I missed that.

That commit is part of 0.50.1 so we can remove this. I’ll do it.

On Jul 28, 2020, at 1:05 PM, Malik Shahzad Muzaffar <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:



@smuzaffar commented on this pull request.

________________________________

In pip/requirements.txt<https://github.com/cms-sw/cmsdist/pull/6108#discussion_r461498315>:

> notebook==6.0.2 ; python_version>'3.0'
-numba==0.50.0dev0 ; python_version>'3.0'
+numba==0.50.1 ; python_version>'3.0'


@davidlange6<https://github.com/davidlange6> , numba 0.50.1 was not picked up due to hard coded commit in the https://github.com/cms-sw/cmsdist/blob/d31f5a3c35fca2d12b62afe01eb6441f6840b126/pip/py3-numba.file .

@mrodozov<https://github.com/mrodozov> , please check if py3-numba-llvm33 patch is needed for numba 0.50.1?

**Comment by davidlange6** [comment]
please test

**Comment by makortel** [comment]
> seems that these recotau differences are also seen in other pull requests that are purely technical. A known issue? @smuzaffar

Perhaps the issue reported here https://github.com/cms-sw/cmssw/pull/30341#discussion_r460444996? (and apparently fixed in https://github.com/cms-sw/cmssw/pull/30919)

**Comment by davidlange6** [comment]
this doesn't look related - have we broken the externals stack (eg, rivet)

**Comment by davidlange6** [comment]
@smuzaffar @mrodozov

**Comment by davidlange6** [comment]
please test

**Comment by smuzaffar** [comment]
+externals
message logger differences in the comparison

**Comment by smuzaffar** [comment]
lets wait for 11.2.0.pre3 to be build (along with its root622 and geant106 flavors)

File: pip/requirements.txt (hunk 1 above)
Comment (mrodozov) [reply pip/requirements.txt:146]: I see this is done. 
anyway, I'm checking out this whole thing to see how it goes with the additional awkward and uproot :/

**Pull request merged**